apollo-router/src/lib.rs

```diff
@@ -70,7 +72,17 @@ impl Subscriber for RouterSubscriber {
         }
```

Comment:
it might be time to move the `RouterSubscriber` implementation outside of `lib.rs`, no?

Reply:
Yeah. It should probably live in `apollo-router-core` as well... I was trying to keep this commit/change simple, but I'll go ahead and do that.

Reply:
Done: bfdba96b403e99eb4b46f4531e46fd3376d7b02a